Cargo.toml

```diff
@@ -9,3 +9,4 @@ documentation = "https://docs.rs/mint"
 keywords = ["math"]
 
 [dependencies]
+serde = { version = "1.0.15", optional = true, features = ["derive"] }
```

Comment:
do you need `1.0.15` specifically? or just `1`?

Reply:
Probably just `1`, I suppose we can trust serde to not break semver.